app/services/platform-apps/api/modules/display.ts

```diff
@@ -0,0 +1,118 @@
+import { Module, apiMethod, EApiPermissions, IApiContext } from './module';
+import { Display } from 'services/video';
+import uuid from 'uuid/v4';
+import electron from 'electron';
+import { Subscription } from 'rxjs/Subscription';
+
+interface IDisplayCreateOptions {
+  position: IVec2;
```

Comment:
when I see `position` I always have a question, is it position of the left-top corner of the center of the screen?

Reply:
Everything in SLOBS uses top-left as the origin currently.